neqo-transport/src/connection/tests/ecn.rs

```diff
@@ -98,8 +98,16 @@ pub fn migration_with_modifiers(
     burst: usize,
 ) -> (IpTos, IpTos, bool) {
     fixture_init();
-    let mut client = new_client(ConnectionParameters::default().max_streams(StreamType::UniDi, 64));
-    let mut server = new_server(ConnectionParameters::default().max_streams(StreamType::UniDi, 64));
+    let mut client = new_client(
+        ConnectionParameters::default()
+            .pmtud(false)
```

Comment:
I think that it would be better if PMTUD were disabled by default.

Reply:
I was hoping that at least some tests wouldn't fail with it enabled by default and so exercise some of the logic, but I can flip the default and do all-new PMTUD tests.